inductiva/tasks/task.py

```diff
@@ -62,37 +63,32 @@ def from_api_info(cls, info: Dict[str, Any]) -> "Task":
 
         return task
 
-    def __enter__(self):
-        """Enter context manager for managing a blocking execution.
+    @contextlib.contextmanager
+    def sync_context(self):
```

Comment:
This has to be used with task.wait() then, is that it?

Reply:
no, it can be used with task.get_output() as well. But to simulate what actually happens in the sync calls, it's with task.wait().

Reply:
How, okay! Then this is actually great! Could you detail then a bit more about the possibilities.